### PR TITLE
Update _workflow-tasks.scss - Changed font-size from px to rem 

### PR DESCRIPTION
**Pull request by IvyJeptoo**
_Please check the following:_

-   [ ] Do the tests still pass?[^1]
-   [ ] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

**Comment by IvyJeptoo** [comment]
@thibaudcolas kindly review my PR

File: client/scss/components/_workflow-tasks.scss [hunk 1]
```diff
@@ -1,30 +1,30 @@
 .workflow-tasks {
-  $task-width: 117px;
-  $task-height: 56px;
+  $task-width: 7.313rem;
```
Comment (thibaudcolas) [inline client/scss/components/_workflow-tasks.scss:2]: This seems off, should be `7.3125rem`. Could you make sure to remove all rounding you’ve done in this file? We don’t want this change to visually alter the appearance of the component.

File: client/scss/components/_workflow-tasks.scss [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 
   list-style-type: none;
 
   &__task {
     display: inline-block;
     background-color: theme('colors.secondary.50');
-    border: 2px solid theme('colors.secondary.100');
-    border-radius: 5px;
+    border: 0.125rem solid theme('colors.secondary.100');
+    border-radius: 0.313rem;
```
Comment (thibaudcolas) [inline client/scss/components/_workflow-tasks.scss:11]: I don’t see why we would want the border width or radius to change depending on the font size. Could you explain why you did this?

File: client/scss/components/_workflow-tasks.scss [hunk 3]
```diff
@@ -1,30 +1,30 @@
 .workflow-tasks {
-  $task-width: 117px;
-  $task-height: 56px;
+  $task-width: 7.313rem;
+  $task-height: 3.5rem;
```
Comment (thibaudcolas) [inline client/scss/components/_workflow-tasks.scss:3]: I’m surprised we’d want to do this in `rem`. Isn’t that going to break the layout?

File: client/scss/components/_workflow-tasks.scss [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
     box-sizing: border-box;
-    width: $task-width;
+    min-width: $task-width;
     height: $task-height;
-    margin: 7px;
-    padding-inline-start: 9px;
-    padding-inline-end: 9px;
+    margin: 0.438rem;
+    padding-inline-start: 0.563rem;
+    padding-inline-end: 0.563rem;
```
Comment (thibaudcolas) [inline client/scss/components/_workflow-tasks.scss:18]: This (and changing the `margin`) below. Also seems superfluous to me. It’d change the layout quite a lot of if we make everything bigger based on the font size?

**Review by thibaudcolas — CHANGES_REQUESTED** [review]
@IvyJeptoo I think this needs a bit more work! Also make sure to:

- Describe what you’ve done. It looks like you submitted this pull request without describing your change, nor following any of our template
- Describe how you’ve tested this so far, and why you chose to make the changes you did.
- Add screenshots. In particular I want you to share what this looks like with a non-default font size set in the browser.

**Comment by lb-** [comment]
@IvyJeptoo  - is everything OK - it looks like you deleted/closed this PR.

**Comment by IvyJeptoo** [comment]
@lb- I deleted it so that i could make changes. I have opened another PR. thank you for the concern

**Pull request closed without merging**